_layouts/search.html

```diff
@@ -55,4 +55,8 @@ <h1>{{ page.t.search.search_results_for }} '<span id="search-term"></span>'</h1>
 </div>
 
 <script src="https://unpkg.com/lunr/lunr.min.js"></script>
+{% if page.language != 'en' %}
+  <script src="https://cdn.jsdelivr.net/npm/lunr-languages@1.4.0/lunr.stemmer.support.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/lunr-languages@1.4.0/lunr.{{ page.language }}.js"></script>
```

Comment:
My only concern with this is whether the language codes match the language codes in `lunr.xx.js` files from `lunr.languages`

Reply:
In that case the script tag will get a 404. There should be no other side effects. Admittedly messy, but at least it's only on the search page.

The bigger problem is that their search likely won't work. I'm not sure yet if there's a way to solve that, other than by implementing a new language in lunr-languages.